flit_core/flit_core/config.py

```diff
@@ -73,6 +74,8 @@ class ConfigError(ValueError):
     'dynamic',
 }
 
+default_license_files_globs = ['COPYING*', 'LICENSE*']
```

Comment:
Would you consider supporting "Licence*"? (The non-US English spelling) 

[Setuptools includes](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html) COPYING, LICENCE, LICENSE, NOTICE, and AUTHORS by default, for context -- I'm not proposing to include the latter two here.

```suggestion
default_license_files_globs = ['COPYING*', 'LICEN[CS]E*']
```

Reply:
Would be fine with me. I just copied the existing values. Seems like they were added in https://github.com/pypa/flit/pull/99#discussion_r97009036

Reply:
Thanks -- I've added another GH suggestion for the other place that would need to change.